core/jobs/batch_jobs/suggestion_stats_computation_jobs.py

```diff
@@ -269,6 +270,564 @@ def _generate_translation_contribution_model(
+            | 'Remove review keys' >> beam.Values()  # pylint: disable=no-value-for-parameter
+            | 'Transform review result to job run result' >> (
+                job_result_transforms.ResultsToJobRunResults())
+        )
+        user_question_contribution_stats_error_job_run_results = (
+            question_contribution_stats_results
+            | 'Filter question contribution err results' >> beam.Filter(
+                lambda key_and_result: key_and_result[1].is_err())
+            # Pylint disable is needed because pylint is not able to correctly
+            # detect that the value is passed through the pipe.
+            | 'Remove question contribution keys' >> beam.Values()  # pylint: disable=no-value-for-parameter
+            | 'Transform question contribution result to job run result' >> (
+                job_result_transforms.ResultsToJobRunResults())
+        )
+
+        unused_contribution_put_result = (
+            user_contribution_stats_models
+            | 'Put contribution models into the datastore' >> ndb_io.PutModels()
+        )
+        unused_review_put_result = (
+            user_review_stats_models
+            | 'Put review models into the datastore' >> ndb_io.PutModels()
+        )
+        unused_question_contribution_put_result = (
+            user_question_contribution_stats_models
+            | 'Put question contribution models into the datastore' >> ndb_io.PutModels()
+        )
+
+        user_stats_models_job_run_results = (
+            user_contribution_stats_models
+            | 'Create contribution job run result' >> (
+                job_result_transforms.CountObjectsToJobRunResult())
+        )
+        user_review_stats_models_job_run_results = (
+            user_review_stats_models
+            | 'Create review job run result' >> (
+                job_result_transforms.CountObjectsToJobRunResult())
+        )
+        user_question_contribution_stats_models_job_run_results = (
+            user_question_contribution_stats_models
+            | 'Create question contribution job run result' >> (
+                job_result_transforms.CountObjectsToJobRunResult())
+        )
+
+        return (
+            (
+                user_stats_error_job_run_results,
+                user_stats_models_job_run_results,
+                user_review_stats_error_job_run_results,
+                user_review_stats_models_job_run_results,
+                user_question_contribution_stats_error_job_run_results,
+                user_question_contribution_stats_models_job_run_results
+            )
+            | 'Merge job run results' >> beam.Flatten()
+        )
+
+    @staticmethod
+    def _generate_translation_stats(
+        suggestions: Iterable[suggestion_registry.SuggestionTranslateContent],
+        opportunity: Optional[opportunity_domain.ExplorationOpportunitySummary],
+        translation_contribution_key_required: bool
+    ) -> Iterator[
```

Comment:
```suggestion
    def _generate_translation_stats(
        suggestions: Iterable[suggestion_registry.SuggestionTranslateContent],
        opportunity: Optional[opportunity_domain.ExplorationOpportunitySummary],
        *,
        translation_contribution_key_required: bool
    ) -> Iterator[
```

Reply:
This is outdated and method has been updated after this suggestion.